README.md

```diff
@@ -0,0 +1,38 @@
+# RPF-Country-Dash
+Dash app for displaying visualizations fro RPF project
+
+## Development
+
+First `git clone` this repo.
+
+Then generate and obtain your access token from databricks following this instruction: https://docs.databricks.com/en/dev-tools/auth/pat.html
+Get connection details for the SQL Warehouse compute resource which is used for providing the database connection: https://docs.databricks.com/en/integrations/compute-details.html
+
+Export the obtained information to the following environment variables:
+
+```bash
+export ACCESS_TOKEN="[access token]"
+export SERVER_HOSTNAME="[server hostname, e.g. adb-12345678.12.azuredatabricks.net"
+export HTTP_PATH="[http path, e.g. /sql/1.0/warehouses/abcdxxx123]"
+```
+
+
+Then to setup and verify the app works locally:
+
+```bash
+pip install -r requirements.txt
+python app.py
+open http://127.0.0.1:8050/
+```
+You should see the data app.
+
+
+
+## Development within docker container
+1. Edit .env to update your environment variables
```

Comment:
Currently .env is tracked by git. Generally if we expect people to edit a file, it should not be tracked by git to avoid constantly having a dangling modified file. A common practice is to have a `.env.example` with the content you currently have in the .env file and then check in the `.env.example` file while add `.env` to `.gitignore`. Then in the README, have instructions like:

## Setup
1. Copy the `.env.example` file to `.env` & update it with the environment varaibles described above:
   ```sh
   cp .env.example .env